brightener.h

```diff
@@ -3,6 +3,6 @@
 
 using namespace std;
 
-shared_ptr<Image> BrightenWholeImage(shared_ptr<Image> inputImage, int& attenuatedPixelCount);
-shared_ptr<Image> AddBrighteningImage(shared_ptr<Image> inputImage, shared_ptr<Image> imageToAdd,
+shared_ptr<Image> BrightenWholeImage(const shared_ptr<Image> inputImage, int& attenuatedPixelCount) ;
```

Comment:
The `const` here says that the `inputImage` pointer cannot change, but the image it's pointing to can change.
Just a comment on the syntax.
It's a minor comment, because in this case, the Image is immutable so nobody can change it anyway.